### DISCUSSION TITLE
[@types/bootstrap] Tab.getOrCreateInstance is missing

### DISCUSSION THREAD

**Discussion by lukasjuhrich** [DefinitelyTyped/DefinitelyTyped#54026] [Issues with a @types package] [answered]
Just like [bootstrap.tab.getInstance(element)](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bootstrap/js/dist/tab.d.ts#L17), there is a static method [`getOrCreateInstance`](https://getbootstrap.com/docs/5.0/components/navs-tabs/#getorcreateinstance). The `types/bootstrap/js/dist/tab.d.ts` needs to be extended by the following declaration:

```typescript
    /**
     * Static method which allows you to get the tab instance associated with a
     * DOM element, or create a new one in case it wasn’t initialised
     */
    static getOrCreateInstance(element: Element): Tab | null;
```

*Edit:* I'm a bit confused by this whole discussions thing, I was under the impression that this created an issue. Feel free to redirect me to the proper place if I did something wrong.

**Comment by darthf1** [comment, marked as answer]
Resolved in `5.1.2`.
